Fakebook.Posts/Fakebook.Posts.UnitTests/CommentTest.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Fakebook.Posts.Domain.Models;
+using Xunit;
+
+namespace Fakebook.Posts.UnitTests
+{
+    public class CommentTest
+    {
+        private const int Id = 1;
+
+        [Fact]
+        public void Comment_Constructor_Pass()
+        {
+            //arrange
+            const string Content = "This is some content";
+            const string userEmail = "damion.silver@test.com";
+            //act
+            Comment newcomment = new Comment(userEmail, Content);
+
+            //assert
+            Assert.NotNull(newcomment);
```

Comment:
it's not a problem, but FYI there's no need for any assertion here, since the result of a ctor can't ever be null. leave it out and you're still just testing that the ctor doesn't throw an exception.

Reply:
I erased the assertion.